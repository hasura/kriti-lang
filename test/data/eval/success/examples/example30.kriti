"{{ $[1].friends[1] }}"