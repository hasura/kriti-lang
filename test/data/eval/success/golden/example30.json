"{\"id\":1,\"name\":\"Charmaine Tate\"}"